frontend.rst

```diff
@@ -1,21 +1,44 @@
-Managing CSS and JavaScript
-===========================
-
-.. admonition:: Screencast
-    :class: screencast
-
-    Do you prefer video tutorials? Check out the `Webpack Encore screencast series`_.
-
-Symfony ships with a pure-JavaScript library - called Webpack Encore - that makes
-it a joy to work with CSS and JavaScript. You can use it, use something else, or
-create static CSS and JS files in your ``public/`` directory directly and
-include them in your templates.
+Symfony Frontend Tools
+======================
```

Comment:
This document is now a bit more important than before. On https://symfony.com/doc/current/index.html, I don't think we link to it at all. After we merge this, we probably should.

Reply:
Good idea. I added some TODO to the PR description for post-merge changes to the main documentation page.